src/main/java/uk/gov/cslearning/record/domain/Learner.java

```diff
@@ -19,7 +16,6 @@
     @Column(nullable = false, length = 60)
     private String uid;
 
-    @ToString.Exclude
```

Comment:
This caused an error as it created a circular reference. Also this isn't used - in the database learner and event are referenced from booking, since we are always accessing this data in the context of a booking.